subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -311,19 +311,39 @@ void ztest_test_fail(void)
 		longjmp(test_suite_fail, 1);
 	case TEST_PHASE_BEFORE:
 	case TEST_PHASE_TEST:
-	case TEST_PHASE_AFTER:
-	case TEST_PHASE_TEARDOWN:
 		PRINT(" at %s function\n", get_friendly_phase_name(phase));
 		longjmp(test_fail, 1);
+	case TEST_PHASE_AFTER:
+	case TEST_PHASE_TEARDOWN:
 	case TEST_PHASE_FRAMEWORK:
-		PRINT("\n");
+		PRINT(" ERROR: cannot fail in test '%s()', bailing\n",
```

Comment:
I think you mean "phase", not "test" here.

Reply:
> I think you mean "phase", not "test" here.

Yeah, I might have meant "test phase".... Would you be ok if I fix it in another PR? We have all the approvals for this one and I don't want to reset the review over my poor grammar.